src/main/java/at/hannibal2/skyhanni/test/command/CopyActionBarCommand.kt

```diff
@@ -0,0 +1,16 @@
+package at.hannibal2.skyhanni.test.command
+
+import at.hannibal2.skyhanni.data.ActionBarData
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.StringUtils.removeColor
+
+object CopyActionBarCommand {
+    fun command(args: Array<String>) {
+        val noFormattingCodes = args.size == 1 && args[0] == "true"
+        val (actionBar, status) = if (noFormattingCodes) Pair(ActionBarData.getActionBar().removeColor(), "without") else Pair(ActionBarData.getActionBar(), "with")
```

Comment:
This line looks horrible. we should maybe use removeColor with parameter doNothing instead

Reply:
I did not make that file, i merged multiple PR into this one, Cal is gonna look at this